code/modules/projectiles/gun.dm

```diff
@@ -192,7 +192,7 @@
 		user << "<span class='warning'>[src]'s trigger is locked. This weapon doesn't have a firing pin installed!</span>"
 	return 0
 
-obj/item/weapon/gun/proc/newshot()
+obj/item/weapon/gun/proc/recharge_newshot()
```

Comment:
name change so guns and ammo casings don't have different procs with the same name.